src/main/java/org/opensrp/repository/postgres/mapper/custom/xml/CustomLocationMetadataMapper.xml

```diff
@@ -217,5 +226,57 @@
 		LIMIT #{limit} OFFSET #{offset}
 
 	</select>
+	<sql id="searchCondition">
+        with locationAndTag as(
+        select
+        lm.id,
+        lm.location_id,
+        lm.parent_id,
+        lm."name",
+        ltm.location_tag_id,
+        lt."name" tag_name
```

Comment:
This may not be required since you can dynamically add properties to the current location object. See https://github.com/OpenSRP/opensrp-server-core/pull/190/files#r413147934